src/redux/books/Books.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD_BOOK = 'bookstore/booksReducer/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/booksReducer/REMOVE_BOOK';
+
+// Reducer
+export default function booksReducer(state = [], action = {}) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return [...state.filter((book) => (book.id !== action.payload))];
+    default: return state;
+  }
+}
+
```

Comment:
- [ ] As you can see below, I cant add or remove. Kindly fix it.
_When I tried adding it produces a blank page and when its refreshed book is not added_

https://user-images.githubusercontent.com/70269507/174943368-8beaa12a-7bd8-41a6-bfff-74753d37074d.mov